applications/nrf_desktop/doc/click_detector.rst

```diff
@@ -1,6 +1,50 @@
 .. _click_detector:
 
-click_detector module
+Click detector module
 #####################
 
-The ``click_detector`` module tracks button events to detect clicks.
+The ``click_detector`` module is used to send ``click_event`` when recognized type of click is recorded for button defined in the module configuration.
+
+Click type
+**********
+
+The module records the following click types:
```

Comment:
Rework the list to the following formatting:
* :cpp:enumerator:`CLICK_SHORT` - Button pressed and released after short time.

Reply:
done